cmd/vicadmin/vicadm.go

```diff
@@ -46,15 +46,20 @@ import (
 	"github.com/vmware/govmomi/vim25/types"
 
 	"github.com/vmware/vic/lib/metadata"
+	"github.com/vmware/vic/lib/pprof"
 	"github.com/vmware/vic/pkg/trace"
 	"github.com/vmware/vic/pkg/vsphere/compute"
 	"github.com/vmware/vic/pkg/vsphere/extraconfig"
 	"github.com/vmware/vic/pkg/vsphere/session"
 	"github.com/vmware/vic/pkg/vsphere/vm"
 )
 
+const (
+	timeout = time.Duration(2 * time.Second)
```

Comment:
I still think we need to bump it, 2 seconds will cause issues with large vc's with esxi's on slow networks.